__tests__/app.test.js

```diff
@@ -127,4 +127,49 @@ describe("GET /api/articles", () => {
         expect(body.msg).toBe("Route not found");
       });
   });
+  test("GET:404 sends an appropriate status and error meaasge when given a valid but non existent id", () => {
+    return request(app)
+      .get("/api/articles/123")
+      .expect(404)
+      .then((res) => {
+        expect(res.body.msg).toBe("article doesn't exist");
+      });
+  });
+});
+
+describe.only("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with an array of comments for the given article_id, sorted by created_at in descending order", () => {
+    return request(app)
+      .get("/api/articles/9/comments")
+      .expect(200)
+      .then(({ body }) => {
+        console.log(body)
+        const { articleComments } = body;
+
+        expect(Array.isArray(articleComments)).toBe(true);
```

Comment:
This assertion becomes redundant as we use a forEach in this test - if it wasn't an array we wouldn't be able to and the test would fail so we don't explicitly need to check that the articles are an array :slightly_smiling_face: